pkgs/top-level/all-packages.nix

```diff
@@ -12336,6 +12336,8 @@ let
 
   pidginmsnpecan = callPackage ../applications/networking/instant-messengers/pidgin-plugins/msn-pecan { };
 
+  pidginmra = callPackage ../applications/networking/instant-messengers/pidgin-plugins/pidgin-mra { };
```

Comment:
Better use `pidgin-mra` for new packages. It's preferable to use dashes, especially if they're used upstream (or in most distros). http://nixos.org/nixpkgs/manual/#sec-package-naming